packages/@ember/-internals/glimmer/lib/modifiers/internal.ts

```diff
@@ -78,6 +78,10 @@ export class InternalModifierManager
     return null;
   }
 
+  getDebugInstance(): unknown {
+    return null;
+  }
+
```

Comment:
```suggestion
```

Reply:
Was hoping I could use this to commit the change directly from the UI, saving a roundtrip, but it wouldn't let me also add a diff for the `abstract class ...` because that line was unchanged 😐

Reply:
this is a required method

```
Class 'InternalModifierManager' incorrectly implements interface 'InternalModifierManager<InternalModifierState, typeof InternalModifier>'.
     Property 'getDebugInstance' is missing in type 'InternalModifierManager' but required in type 'InternalModifierManager<InternalModifierState, typeof InternalModifier>'. [2420]
```

Reply:
nevermind: https://github.com/emberjs/ember.js/pull/20658/commits/a13c6275687d119a8de0dd9cb0f9755df2b8ba4d#r1544906209